specification/compute/Compute.Management/models.tsp

```diff
@@ -4115,6 +4283,20 @@ model OrchestrationServiceSummary {
    */
   @visibility(Lifecycle.Read)
   serviceState?: OrchestrationServiceState;
+
+  /**
+   * The latest operation status of the service. Minimum API version for this property is 2025-04-01.
+   */
+  @visibility(Lifecycle.Read)
+  @added(Versions.v2025_04_01)
+  latestOperationStatus?: OrchestrationServiceOperationStatus;
+
+  /**
+   * The last time when the operation status changed. Minimum API version for this property is 2025-04-01.
+   */
+  @visibility(Lifecycle.Read)
+  @added(Versions.v2025_04_01)
+  lastStatusChangeTime?: utcDateTime;
```

Comment:
Let the name or description say its in UTC.

Reply:
added to description